helper/include/odp/helper/table.h

```diff
@@ -91,7 +91,11 @@ extern "C" {
  * @def ODPH_TABLE_NAME_LEN
  * Maximum length of table name, including the null character
  */
-#define ODPH_TABLE_NAME_LEN      32
+/*
+ * Minimum of ODP_SHM_NAME_LEN, (ODP_QUEUE_NAME_LEN - 3),
+ * (ODP_POOL_NAME_LEN - 8).
+ */
+#define ODPH_TABLE_NAME_LEN 24
```

Comment:
I guess you could do the calculations explicitly here, also elaborating the meaning of "3" and "8" could be helpful. Can you actually infer the numbers based on some constants so they don't have to be magic numbers?

Reply:
Those constants don't currently exist, and even if they did, they'd probably be defined in the headers of specific table types, which we probably wouldn't want to include from here. I.e. there's an awkward dependency from this generic table name length to name lengths in different table types. In this commit, that dependency is implied by this comment, I didn't want to add an actual compile time dependency.

Just by searching for ODPH_TABLE_NAME_LEN, it's not hard to find that there's usage such as ODPH_TABLE_NAME_LEN + 3 in cuckootable and ODPH_TABLE_NAME_LEN + 8 in iplookuptable, where the magic numbers come from. But I'll try to improve the comment.

This header is anyway being removed, these are just minimal changes to fix the known potential problems before that happens.

Reply:
Comment revised in v3. (I'm not sure whether it's better or worse.)

Reply:
It's pretty informative now, resolving this.